### ISSUE TITLE
[Bug]: embed readme.md is added into symbols.nupkg when packing 

### ISSUE THREAD

**Issue by heng-liu** [NuGet/Home#11634]
### NuGet Product Used

dotnet.exe

### Product Version

6.0.200

### Impact

Other

### Repro Steps & Context

1. Create a project as following:
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net6.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
    <PackageReadmeFile>README.md</PackageReadmeFile>
    <IncludeSymbols>true</IncludeSymbols>
    <SymbolPackageFormat>symbols.nupkg</SymbolPackageFormat>
  </PropertyGroup>

  <ItemGroup>
    <None Include="docs\README.md" Pack="true" PackagePath="\"/>
  </ItemGroup>
</Project>
```
2. Create an docs\README.md file
3. Run `dotnet pack`
4. Check the symbols.nupkg, the README.md is in the symbol package.

BTW, the README.md is not added when the symbol format is snupkg.

**Comment by heng-liu** [comment]
Thanks for @chgill-MSFT 's help on confirming that this is not the expected experience.

**Comment by nkolev92** [comment]
Does having a readme.md in a package fail pushing the snupkgs to nuget.org?

**Comment by nkolev92** [comment]
@heng-liu Does having a readme.md in a package fail the pushing of the snupkgs to nuget.org?

**Comment by nkolev92** [comment]
Reviewing this issue again, are there any side-effects from readme.md being in the symbols.nupkg? 
To the best of my knowledge, there's no validation, and this does not cause any legal concern (compared to say license.md). 

Isn't this just a thing? And maybe something not relevant enough to fix.

Looking for @heng-liu and @chgill-MSFT's opinions.

**Comment by heng-liu** [comment]
Sorry but I didn't find anything in docs..
Hi @chgill-MSFT , do you have any idea if this issue is a blocker for https://github.com/NuGet/NuGet.Client/pull/4476? Thanks!

**Comment by nkolev92** [comment]
I don't think it can be a blocker for https://github.com/NuGet/NuGet.Client/pull/4476 as there's no validation on the symbols.nupkg side for what files in addition to the pdb are allowed in there. 
For example, we've icon.png in there for a while and we haven't had issue. I can only imagine it's the same thing.

**Comment by heng-liu** [comment]
I see.. Thanks for the clarification! I'll merge the PR then.

**Comment by nkolev92** [comment]
Team Triage; We think that there'll be no real impact from this, so we shouldn't need to prioritize it
